ott/core/quad_problems.py

```diff
@@ -189,14 +193,17 @@ def is_low_rank(self) -> bool:
 
   @property
   def linear_loss(self) -> Tuple[Loss, Loss]:
+    """Linear part of the GW loss."""
     return self.loss.f1, self.loss.f2
 
   @property
   def quad_loss(self) -> Tuple[Loss, Loss]:
+    """Quadratic part of the GW loss."""
     return self.loss.h1, self.loss.h2
 
   @property
   def is_balanced(self) -> bool:
+    """Whether the problem is balanced."""
     return ((not self.gw_unbalanced_correction) or
```

Comment:
side comment: the fact that `gw_unbalanced_correction` is set to False seems to trigger the fact this is not balanced, regardless of `tau_a` and `tau_b`. Should this be a `and` ?

Reply:
If tau_a and tau_b are both equal to 1, then is_balanced return True, which is as expected. 

If tau_a and tau_b are not equal to 1, there are two cases. If there is no correction, `not self.gw_unbalanced_correction` is True, and in this case, it behaves as the balanced case (which is as expected as we ignore the unbalanced correction and the rescaling), if there is a correction, (not self.gw_unbalanced_correction is False) then the is_balanced function returns False. The logic is good I think.